docker-compose.background.yml

```diff
@@ -1,25 +1,37 @@
-version: '3'
+version: "3"
 services:
-  validator_service:
-    image: infernocommunity/fhir-validator-service
-    volumes:
-      - ./lib/deqm_test_kit/igs:/home/igs
-  fhir_validator_app:
-    image: infernocommunity/fhir-validator-app
-    depends_on:
-      - validator_service
+  # validator_service:
```

Comment:
Do we need to hold on to these commented out sections with the previous validator config?